pkg/operator/controllers/machine/machine_controller.go

```diff
@@ -22,18 +22,20 @@ import (
 )
 
 type MachineReconciler struct {
-	maocli                 maoclient.Interface
-	arocli                 aroclient.Interface
-	log                    *logrus.Entry
+	log *logrus.Entry
+
+	arocli aroclient.Interface
+	maocli maoclient.Interface
+
 	isLocalDevelopmentMode bool
 	role                   string
 }
 
-func NewMachineReconciler(log *logrus.Entry, maocli maoclient.Interface, arocli aroclient.Interface, isLocalDevelopmentMode bool, role string) *MachineReconciler {
+func NewReconciler(log *logrus.Entry, arocli aroclient.Interface, maocli maoclient.Interface, isLocalDevelopmentMode bool, role string) *MachineReconciler {
```

Comment:
Unless there is ambiguity (multiple reconcilers in same package), we just use the name `NewReconciler`.